mesonbuild/interpreter/interpreter.py

```diff
@@ -129,6 +129,7 @@
 
     ProgramVersionFunc = T.Callable[[T.Union[ExternalProgram, build.Executable, OverrideProgram]], str]
 
+    TestClass = T.TypeVar('TestClass', Test)
```

Comment:
doesn't this need to be `bound=Test`?

Reply:
*/me hides* - it seems to work but my `mypy`-fu is not strong